tests/pdctl/keyspace/keyspace_group_test.go

```diff
@@ -120,14 +121,38 @@ func TestSplitKeyspaceGroup(t *testing.T) {
 	leaderServer := tc.GetServer(tc.GetLeader())
 	re.NoError(leaderServer.BootstrapCluster())
 
+	var splitKeyspaces []string
+	for i := 1; i <= 130; i++ {
+		splitKeyspaces = append(splitKeyspaces, fmt.Sprintf("%d", i))
+	}
 	// split keyspace group.
 	testutil.Eventually(re, func() bool {
-		args := []string{"-u", pdAddr, "keyspace-group", "split", "0", "1", "2"}
+		args := append([]string{"-u", pdAddr, "keyspace-group", "split", "0", "1"}, splitKeyspaces...)
 		output, err := pdctl.ExecuteCommand(cmd, args...)
 		re.NoError(err)
 		return strings.Contains(string(output), "Success")
-	}, testutil.WithWaitFor(20*time.Second))
```

Comment:
It will become unstable due to timeout

Reply:
Maybe you can check the default value. If there is a timeout. We need to investigate why instead of enlarging the wait time.

Reply:
Because there are more keyspaces than previous, it will take more time to prepare and split.

Reply:
So, how long will it cost if we have 500 keyspace groups?